aspnetcore/fundamentals/app-state/3.0samples/RazorPagesContacts/Data/RazorPagesContactsContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using RazorPagesContacts.Models;
+
+namespace RazorPagesContacts.Data
+{
+    public class RazorPagesContactsContext : DbContext
+    {
+        public RazorPagesContactsContext (DbContextOptions<RazorPagesContactsContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<RazorPagesContacts.Models.Customer> Customer { get; set; }
```

Comment:
Simplify, since the namespace is already imported at the top of the file.

```suggestion
        public DbSet<Customer> Customer { get; set; }
```